rootfs/api/tests/test_app.py

```diff
@@ -28,6 +28,10 @@ def _mock_run(*args, **kwargs):
     return [0, 'mock']
 
 
+def _mock_port(*args, **kwargs):
```

Comment:
Could `_mock_port` be moved into tests/__init__.py? That would be less repetitive.